src/Stryker.Core/Stryker.Core.UnitTest/Mutators/StringMutatorTests.cs

```diff
@@ -32,5 +32,36 @@ public void ShouldMutate(string original, string expected)
                 .Token.Value.ShouldBe(expected);
             mutation.DisplayName.ShouldBe("String mutation");
         }
+
+        [Fact]
+        public void ShouldNotMutateOnRegexExpression()
+        {
+            var expressionSyntax = SyntaxFactory.ParseExpression("new Regex(\"myregex\")");
```

Comment:
Maybe also test if this works when fully specifying the namespace like: 
```suggestion
            var expressionSyntax = SyntaxFactory.ParseExpression("new System.Text.RegularExpressions.Regex(\"myregex\")");
```